frontend/src/components/FormReactHookCreateNewInquiry.js

```diff
@@ -31,22 +31,22 @@ export default function FormReactHookCreateNewInquiry({onAdd}) {
                        ref={register({required: true})}/>
                 {errors.partDescription && "Required"}
 
-                <label>Maximum Length of Part [mm]</label>
+                <label>Max Length [mm]</label>
                 <input name="length"
                        ref={register({required: true})}/>
                 {errors.length && "Required"}
```

Comment:
you could think about creating an Input Field Component. But I would do it in a separate pull request